setup.py

```diff
@@ -27,16 +27,14 @@
     "ipython>=7",
     "umap-learn>=0.3.7",
     "seaborn>=0.9.0",
-    "leidenalg>=0.7.0",
-    "python-igraph>=0.7.1",
     "hyperopt>=0.1.2",
 ]
 
 setup_requirements = ['pytest-runner', ]
 
 test_requirements = ['pytest', ]
 
-extras_requirements = {'test': ['scanpy', 'louvain', ]}
+extras_requirements = {'test': ['scanpy', 'louvain', "leidenalg>=0.7.0", "python-igraph>=0.7.1", ]}
```

Comment:
Do we even need to list them as extra requirements now that you've removed `utils.py`?

Reply:
I think we need them to make work scanpy but not sure exactly

Reply:
scanpy really should include it's own dependencies, but whatever, let's close this. Thanks for making the changes.